packages/suite-web/e2e/support/types/index.ts

```diff
@@ -0,0 +1,10 @@
+import { SuiteAnalyticsEvent } from '@trezor/suite-analytics';
```

Comment:
should go to dev dependencies?

Reply:
![Screenshot 2023-06-20 at 13 38 19](https://github.com/trezor/trezor-suite/assets/33235762/38759425-46d4-4525-95d5-3af51bc79a5d)
added